src/components/MaStack/MaStack.vue

```diff
@@ -11,12 +14,14 @@ const alignment = {
   right: 'flex-end',
 }
 
+type StackAlignment = keyof typeof alignment
```

Comment:
I'd rather have the values here, but it might be just a personal preference.

```suggestion
type StackAlignment = 'fill' | 'right'
```